### ISSUE TITLE
No auto reconnect after DDNS IP change

### ISSUE THREAD

**Issue by DavidMStraub** [owntracks/android#394]
This is possibly related to #255:
I have set up a DDNS domain for my MQTT broker, which is only reachable via IPv6. Everything works fine, except every 24h, when the IP (v6) changes, the android app loses connection (failed to connect ... after 30000 ms) and does not reconnect automatically. Note that my DDNS might take a couple of minutes (or even half an hour) to update the IP. Manually clicking on the checkmark in the connection settings, it immediately connects again without problems. I logged the MQTT and found that *no* lwt is published by the broker at any time.

I am using the app v0.6.14 on Android 6.0.1.

**Comment by binarybucks** [comment]
The current version does not reconnect correctly. This will hopeful be
fixed in the next version.

You're not seeing a LWT because the default keepalive interval is 1h.
Mosquitto will consider the client as dead after 1.5x the keepalive
interval after which a LWT will be sent.

--
Alexander Rust

Am 16.08.2016 um 11:11 schrieb David Straub <notifications@github.com>:

This is possibly related to #255
<https://github.com/owntracks/android/issues/255>:
I have set up a DDNS domain for my MQTT broker, which is only reachable via
IPv6. Everything works fine, except every 24h, when the IP (v6) changes,
the android app loses connection (failed to connect ... after 30000 ms) and
does not reconnect automatically. Note that my DDNS might take a couple of
minutes (or even half an hour) to update the IP. Manually clicking on the
checkmark in the connection settings, it immediately connects again without
problems. I logged the MQTT and found that *no* lwt is published by the
broker at any time.

I am using the app v0.6.14 on Android 6.0.1.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/owntracks/android/issues/394>, or mute the thread
